src/webui/btjson.cpp

```diff
@@ -800,6 +801,7 @@ QVariantMap toMap(BitTorrent::TorrentHandle *const torrent)
     ret[KEY_TORRENT_NUM_INCOMPLETE] = torrent->totalLeechersCount();
     const qreal ratio = torrent->realRatio();
     ret[KEY_TORRENT_RATIO] = (ratio > BitTorrent::TorrentHandle::MAX_RATIO) ? -1 : ratio;
+    ret[KEY_TORRENT_STATUS] = torrent->state().toString();
```

Comment:
Thinking back I remember that the call to `this.getRowValue()` uses the name of the column to retrieve the value. In this case the name of the column is `status` so it uses `status` when retrieving the value from the JSON object.